yarn.lock

```diff
@@ -5391,7 +5391,7 @@ axe-core@=4.7.0:
   resolved "https://registry.npmjs.org/axe-core/-/axe-core-4.7.0.tgz"
   integrity sha512-M0JtH+hlOL5pLQwHOLNYZaXuhqmvS8oExsqB1SBYgA4Dk7u/xx+YdGHXaK5pyUfed5mYXdlYiphWq3G8cRi5JQ==
 
-axios-retry@^4.0.0, axios-retry@^4.3.0:
+axios-retry@^4.3.0:
```

Comment:
does ky use axios under the hood? are we still using axios anywhere?

Reply:
https://github.com/ar-io/network-portal/blob/241f0d19e4730f869d5b8418fabf62076aa9af54/yarn.lock#L36-L37 it's nested all over

Reply:
ky does not, looks like it's getting pulled in through ar-io-sdk or one of the other dependencies